docs/docs/tutorials/counter.md

```diff
@@ -0,0 +1,232 @@
+---
+sidebar_position: 3
+description: Build a simple "Counter" application.
+---
+
+# Counter 🎲
+
+:::info
+**Difficulty**: 🟢 Beginner<br/>
+**Length**: 15 minutes
+
+Before getting started, [read the prerequisites](/docs/overview#prerequisites) to make sure your development environment is ready.
+:::
+
+## Overview
+
+In this tutorial, we're going to build an app that exposes a single endpoint and responds with the number of times the route was requested.
+
+When we're done, we should be able to make multiple `GET` requests to the `/` endpoint:
+
+```bash
+# 1st time
+curl --request GET \
+  --url http://localhost:8080
+
+# 2nd time
+curl --request GET \
+  --url http://localhost:8080
+```
+
+And our server should respond with the following responses:
+
+```
+HTTP/1.1 200 OK
+Connection: close
+Content-Length: 21
+Content-Type: text/plain; charset=utf-8
+
+
+You have requested this route 1 time(s).
+
+---
+
+HTTP/1.1 200 OK
+Connection: close
+Content-Length: 21
+Content-Type: text/plain; charset=utf-8
+
+
+You have requested this route 2 time(s).
+```
+
+## Creating a new app
+
+To create a new Dart Frog app, open your terminal, `cd` into the directory where you'd like to create the app, and run the following command:
+
+```bash
+dart_frog create counter
+```
+
+You should see the following output:
```

Comment:
The timing can be different so I would phrase this more like, you should see an output similar too: